config/env/index.js

```diff
@@ -1,5 +1,70 @@
-exports.development = require('./development');
-exports.staging = require('./staging');
-exports.local = require('./local');
-exports.production = require('./production');
-exports.testing = require('./testing');
+const winston = require('winston');
+const DailyRotateFile = require('winston-daily-rotate-file');
+const { hidFormatter } = require('../logs');
+
+const env = process.env.NODE_ENV || 'local';
+const databaseUri = process.env.DATABASE || 'mongodb://db:27017/local';
+const defaultStore = databaseUri.split('/').pop();
+const logLevel = process.env.LOG_LEVEL || 'debug';
+
+const loggerInfo = new DailyRotateFile({
+  name: 'info-file',
+  filename: 'trails/info.log',
+  level: logLevel,
+  timestamp: true,
+});
+
+const loggerError = new DailyRotateFile({
+  name: 'error-file',
+  filename: 'trails/error.log',
+  level: logLevel,
+  timestamp: true,
+});
+
+const loggerConsole = new winston.transports.Console();
+
+const loggerLocal = new winston.transports.File({
+  filename: '/var/log/local.log',
+  timestamp: true,
+});
+
+const transports = [];
+
+if (env == 'local') {
```

Comment:
```suggestion
if (env === 'local') {
```

Reply:
yeah eslint complained about this as well. fix incoming.